pkg/store/topo/store_test.go

```diff
@@ -111,6 +115,14 @@ func TestTopoStore(t *testing.T) {
 	topoEvent = nextEvent(t, ch)
 	assert.Equal(t, topoapi.ID("o1"), topoEvent.ID)
 
+	// Verify the attribute values
+	obj2g, err := store1.Get(context.TODO(), obj2.ID)
+	assert.NoError(t, err)
+	loc := topoapi.GetAttribute(obj2g, "foo", &topoapi.Location{}).(*topoapi.Location)
```

Comment:
Can you also test Labels and see you can set them and you can get the values?  @tomikazi